### ISSUE TITLE
`cy.scrollIntoView` doesn't respect css `scroll-margin` or `scroll-padding` properties

### ISSUE THREAD

**Issue by crfrolik** [cypress-io/cypress#29622]
### Current behavior

If you have an element inside of a scroll container, and you apply something like:

```css
.elem {
  scroll-margin-top: 50px;
}
```

This will work fine if you actually call the element's `scrollIntoView` function. The native function will respect the scroll margin.

However, `cy.scrollIntoView` uses the `jquery.scrollTo` plugin, which doesn't respect these new properties. It hasn't been updated in years, so the only option is to probably abandon this plugin and instead use native `scrollIntoView`.

### Test code to reproduce

See above.

### Cypress Version

13.6.6

### Node version

v20

### Operating System

N/A

**Comment by jennifer-shehane** [comment]
@crfrolik Yes, updating/replacing jquery is a big undertaking. It'd be nice to have a reproducible example to run to show this issue. We'd need to make one to write a failing test anyway.

**Comment by crfrolik** [comment]
> @crfrolik Yes, updating/replacing jquery is a big undertaking. It'd be nice to have a reproducible example to run to show this issue. We'd need to make one to write a failing test anyway.

Thanks for the response.

To be clear, I didn't want to suggest replacing jquery. Just the `jquery.scrollTo` plugin. That may be no small undertaking either, but it's definitely a much smaller surface area than all of jquery.

**Comment by jennifer-shehane** [comment]
Yah we'd be open to that

**Comment by crfrolik** [comment]
@jennifer-shehane I created a reproducible example here:
https://github.com/crfrolik/cypress-scroll-test

**Comment by crfrolik** [comment]
> This issue has not had any activity in 180 days. Cypress evolves quickly and the reported behavior should be tested on the latest version of Cypress to verify the behavior is still occurring. It will be closed in 14 days if no updates are provided.

This is still an unresolved issue.
